src/qt_gui/mainframe.hpp

```diff
@@ -32,7 +32,9 @@ Q_OBJECT
 
     inline static const char* M64P_PLUGIN_PATH{"../emulator/mupen64plus/"},
                             * M64P_CONFIG_SUB_PATH{"config/mupen64plus/config/"},
-                            * M64P_DATA_SUB_PATH{"config/mupen64plus/data/"};
+                            * M64P_DATA_SUB_PATH{"config/mupen64plus/data/"},
+                            * M64P_USER_CONFIG_SUB_PATH{ "config/mupen64plus/config.json" };
```

Comment:
This path is wrong. It should point to "config/config.json". 
Please also rename it to something more fitting, like `MAIN_CONFIG_FILE` or so.
And again please remove the spaces around the string.